src/strings.jl

```diff
@@ -25,7 +25,8 @@ const esriid2code = Dict(
   "World_Behrmann" => ESRI{54017},
   "World_Cylindrical_Equal_Area" => ESRI{54034},
   "World_Robinson" => ESRI{54030},
-  "World_Winkel_Tripel_NGS" => ESRI{54042}
+  "World_Winkel_Tripel_NGS" => ESRI{54042},
+  "ETRF2000-PL / CS92" => EPSG{2180}
```

Comment:
```suggestion
  "ETRF2000-PL_CS92" => EPSG{2180}
```
The ESRI ID is the first string after `PROJCS`:
![image](https://github.com/user-attachments/assets/a7416ef7-3c26-4ba9-9b61-927bb70abc8d)